examples/crd/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build the manager binary
```

Comment:
don't think we should have the deployment stuff here -- this should be only code.  Deployment examples can live in KB

Reply:
Ack